lib-impl/src/main/java/org/ergoplatform/appkit/impl/InputBoxImpl.java

```diff
@@ -63,6 +64,11 @@ public String toJson(boolean prettyPrint) {
         return json;
     }
 
+    @Override
+    public Map<ErgoBox.NonMandatoryRegisterId, ? extends Values.EvaluatedValue<? extends SType>> getRegisters() {
```

Comment:
This sigma types should not be exposed to Java interfaces. And returning registers as Map doesn't really make sense, since they are actually packed into dense array. So I suggest this method to return `List<ErgoValue<?>>`.
To also be consistent with `OutBoxBuilder registers(ErgoValue<?>... registers)`.

Reply:
I believe this is addressed in PR #23

Reply:
That is strange that I don't see my getRegisters method here.